src/py42/util.py

```diff
@@ -72,3 +72,12 @@ def format_dict(dict_, label=None):
     if label:
         return u"{} {}".format(label, indented_dict)
     return indented_dict
+
+
+def filter_attributes(cls):
```

Comment:
I wonder why this was originally called `filter_attributes()`. Just now realizing that something like `get_attributes_from_class()` would be a more appropriate name.

Reply:
Do you mind making this change for this helpful internal util method?

Reply:
No problem. I'm on it.

Reply:
Or actually how about `get_attribute_keys_from_class()` ?